ReactWindows/ReactNative/ReactNative.csproj

```diff
@@ -131,38 +110,15 @@
     <Compile Include="Animated\ValueAnimatedNode.cs" />
     <Compile Include="Bridge\BridgeBusyEventArgs.cs" />
     <Compile Include="Bridge\BridgeIdleEventArgs.cs" />
-    <Compile Include="Bridge\IAsyncCancelable.cs" />
-    <Compile Include="Bridge\IAsyncDisposable.cs" />
-    <Compile Include="Bridge\ILifecycleEventListener.cs" />
-    <Compile Include="Bridge\IPromise.cs" />
-    <Compile Include="Bridge\ReactInstance.cs" />
     <Compile Include="Bridge\CompiledReactDelegateFactory.cs" />
+    <Compile Include="Bridge\ReactInstance.cs" />
     <Compile Include="Bridge\DispatcherHelpers.cs" />
```

Comment:
Super nitpick, this include was moved for no reason ;-).  Doesn't matter for this change, because the .csproj is in serious flux, but for future reference in cases where diffs on the .csproj are smaller its courtesy to keep these kinds of diffs small (they make it less likely that you'll need to merge changes in this file).